docs/ChangeLog.rst

```diff
@@ -5,6 +5,19 @@ Release Notes
 Version 1.0 (in development)
 ============================
 
+Main changes:
+-------------
+* ``Browser.submit`` and ``StatefulBrowser.submit_selected`` accept a larger
+  number of keyword arguments. Arguments are forwarded to
+  `requests.Session.request <http://docs.python-requests.org/en/master/api/#requests.Session.request>`__.
+  [`#166 <https://github.com/MechanicalSoup/MechanicalSoup/pull/166>`__]
+
+Internal changes:
+-----------------
+* Private methods ``Browser._build_request`` and ``Browser._prepare_request``
+  have been replaced by a single method ``Browser._request``.
+  [`#166 <https://github.com/MechanicalSoup/MechanicalSoup/pull/166>`__]
```

Comment:
I usually omit these internal changes from the ChangeLog. I've added "Internal changes" section for other releases for things we should be proud of like 100% coverage and that didn't fit elsewhere, or stuff that developers should be aware of like changes in the way to launch testsuite. But it doesn't harm to have them.

Reply:
The only reason I added these was because there is no truly private method in python, so someone could have been using them. I have no problem with this item being omitted from the ChangeLog. Please feel free to do so if that's your preference!